docs/pages/faq.mdx

```diff
@@ -63,6 +63,19 @@ are not logged in to Teleport or may not have access to a browser.
 Make sure that all Teleport components are at least at version 10.3.0. Older versions
 don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.
 
+## `tsh` is very slow on Windows, what to do?
+
+This may be because user lookups take a long time in your environment. Make sure `tsh` is at least
```

Comment:
I think we've only observed this with AD-joined hosts. We should probably mention that here.

Reply:
Done